assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -58,11 +58,16 @@ function FontControls({ properties, state, setState }) {
             const fontWeightsArr = currentFontWeights.map(
               ({ value: weight }) => weight
             );
-            const newFontWeight =
+            let newFontWeight =
               fontWeightsArr &&
               fontWeightsArr.includes(state.fontWeight.toString())
                 ? state.fontWeight
                 : 400;
+
+            // If the font doesn't have 400 as an option, let's take the first available option.
+            if (!newFontWeight && fontWeightsArr.length) {
+              newFontWeight = fontWeightsArr[0];
+            }
```

Comment:
We need to use `fontWeightsArr.includes('400')` somewhere here for the comment to hold true (_right now `newFontWeight` is always truthy_).

Reply:
Ah, of course, ha! 🤦‍♀Not sure how it ended up like this, thanks! _Changing_

Reply:
Should be addressed in 4860fad where it's checked that the fonts' array would include the assigned weight.

Reply:
`fontWeightsArr.includes('400')` in place of `400` would do the trick?

Reply:
Refactored a bit more in 513e50c

Reply:
Nice, just a note: for `Molle` it will produce `400`, can we live with that?

Reply:
Hmm, right. Perhaps we could do something like
`let defaultWeight = fontWeightsArr?.length ? 400 : '';` to set the default weight empty if the array is empty.

Reply:
@miina Given the back and forth here (and the length of the `onChange` function, it might be worth extracting this logic and adding tests for it. WDYT? Perhaps you could move it below `handleNumberChange` for now, and in the future we can look into the testing part.

Reply:
`''` might uncover some other edge case here:
https://github.com/google/web-stories-wp/blob/588807acb6675c0eeae568ef6b17ccf28254406a/assets/src/edit-story/components/panels/textStyle/font.js#L86